Tests iOS/HomeTests.swift

```diff
@@ -55,8 +55,6 @@ class HomeTests: XCTestCase {
         }
 
         try server.start()
-
-        app.launch()
```

Comment:
curious - why did we decide to remove this and add it to the individual tests?

Reply:
for certain Home tests, we want use different responses than those set up in `setUp` (e.g we want the initial "My List" sync to return a different list for recent saves in web view). if we launch the app during `setUp`, though, our sync requests will already be fulfilled with the responses assigned in `setUp` since the app is launched (initiating the initial sync request chain, for example, for home, my list, etc) and started before our tests are run (after the app is launched, and already loaded data), where we attempted to override the responses. it's similar to a "race condition", where our app is already loaded and running before a test sets up its expected API responses.

Reply:
got it, thank you!